optimoveapi/api/customers.py

```diff
@@ -1,6 +1,12 @@
+import logging
+
 from ..utils import date_to_str
 
 
+logger = logging.getLogger(__name__)
+logging.basicConfig(level=logging.WARNING)
```

Comment:
We should have single place where config for logging is defined for library. I think we can fallow Kenneth's pattern http://docs.python-guide.org/en/latest/writing/logging/

(I know - I added same logging into transport - totally forgot about to fix it)